draft-ietf-quic-recovery.md

```diff
@@ -348,6 +348,13 @@ acknowledgement delays are likely to be non-repeating and limited to the
 handshake. The endpoint can therefore use them without limiting them to the
 max_ack_delay, avoiding unnecessary inflation of the RTT estimate.
 
+Note however that a large acknowledgement delay can result in a substantially
```

Comment:
```suggestion
A large acknowledgement delay can result in a substantially
```

Reply:
This connects it to the previous para, which is useful I think.